signer/src/storage/model.rs

```diff
@@ -156,6 +156,17 @@ pub struct WithdrawalRequest {
     pub sender_address: StacksPrincipal,
 }
 
+impl WithdrawalRequest {
+    /// Return the identifier for the withdrawal request.
+    pub fn request_identifier(&self) -> WithdrawalRequestId {
```

Comment:
i would probably have just called this `request_id()` since that's what it's called everywhere else

Reply:
Well, I felt that would be confusing, since there is a `request_id` field, and `request_id` here usually refers to the one generated by smart contract. I'm not thrilled with the name here though, so I'm definitely open to suggestions.

Reply:
Maybe I should rename the struct too.

Reply:
Aha okay I just quickly assumed it was referring to the same request id i see everywhere else, if it's not then yeah i agree this is confusing 😊

Reply:
QualifiedRequestId or something along those lines maybe?

Reply:
That works! Using that.

Reply:
https://github.com/stacks-network/sbtc/pull/529/commits/dafedbe7cd474fb34390e718f5198bffc04ff2ed

Reply:
Looks nice!